pagefaultgames/pokerogue
# [QoL] Migrate `touch-controls` file to typescript

## What are the changes?


I've migrated the `touch-controls` file to typescript

## Why am I doing these changes?

The whole code-base is in TS and it seems rather off to have this file in JS.




## What did change?



`src/touch-controls.js` is now `src/touch-controls.ts` with all the necessary changes that come with using TS. 

### Screenshots/Videos




https://github.com/pagefaultgames/pokerogue/assets/50131232/6df21133-38f8-46cd-a829-047691978fad

<img width="208" alt="image" src="https://github.com/pagefaultgames/pokerogue/assets/50131232/527e3565-848f-4252-8c1a-b228387d645f">

## How to test the changes?





Run the local game with a touch-device and see if all the touch controls still work. 

## Checklist
